demoscene/utils/groklinks.py

```diff
@@ -225,8 +225,8 @@ class AmpAuthor(BaseUrl):
 class CsdbScener(BaseUrl):
 	canonical_format = "http://csdb.dk/scener/?id=%s"
 	tests = [
-		querystring_match(r'https?://noname\.c64\.org/csdb/scener/', 'id', re.I),
-		querystring_match(r'https?://(?:www\.)?csdb\.dk/scener/', 'id', re.I),
+		querystring_match(r'https?://noname\.c64\.org/csdb/scener/(index.php)?', 'id', re.I),
```

Comment:
I think this (and the others) should be:
```suggestion
		querystring_match(r'https?://noname\.c64\.org/csdb/scener/(?:index\.php)?', 'id', re.I),
```

to tell it to ignore the item when returning substrings (and also not treat the `.` as a wildcard). I don't think it makes a difference with `querystring_match` anyhow, but good to be consistent...

Reply:
Good catch, I amended the commit.